src/main/java/Chat/tasks/Evelyn.java

```diff
@@ -0,0 +1,261 @@
+package Chat.tasks;
+
+import Chat.exceptions.RepeatMark;
+import Chat.exceptions.RepeatUnmark;
+import Chat.exceptions.InvalidIndex;
+
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.Scanner;
+
+import java.util.ArrayList;
+import java.io.File;
+
+
+public class Evelyn {
+    public static ArrayList<Task> tasks;
+    public static int indexOfTask = 0;
+
+    public static void markTask(int index, boolean done) throws InvalidIndex, RepeatMark, RepeatUnmark {
+        if(index < 0 || index >= indexOfTask){
+            throw new InvalidIndex();
+        }
+
+        if (done) {
+            try {
+                tasks.get(index).markAsDone();
+                System.out.println("Nice! I've marked this task as done:");
+            } catch(RepeatMark e){
+                System.out.println("This task is already marked as done.");
+            }
+        } else {
+            try {
+                tasks.get(index).markAsNotDone();
+                System.out.println("OK, I've marked this task as not done yet:");
+            } catch(RepeatUnmark e){
+                System.out.println("This task is already unmarked as not done.");
+            }
+        }
+        System.out.println("  " + tasks.get(index).getStatusIcon() + " "
+                               + tasks.get(index).description);
+        printLine();
+
+    }
+
+    public static void printList(ArrayList<Task> tasks){
+        int index = 1;
+        for (Task task : tasks) {
+            if (task == null) {
+                printLine();
+                break;
+            } else {
+                System.out.println(index + ". " + tasks.get(index - 1).toString());
+                index++;
+            }
+        }
+    }
+
+    public static void echo() {
+        String line;
+        System.out.println("type your command: ");
+        Scanner in = new Scanner(System.in);
+        line = in.nextLine().trim();
+        printLine();
+        if (line.equals("bye")) {
+            return;
+        } else if (line.equals("list")) {
+                System.out.println("Here are the tasks in your list:");
+                printList(tasks);
+
+        } else if (line.startsWith("mark")) {
+            mark(line);
+
+        } else if (line.startsWith("unmark")) {
+            unmark(line);
+
+        } else if (line.startsWith("todo")) {
+            todo(line);
+        } else if (line.startsWith("deadline")) {
+            deadline(line);
+
+        } else if (line.startsWith("event")) {
+            event(line);
+        } else if(line.startsWith("delete")){
+            delete(line);
+        } else {
+            System.out.println("Please enter the correct command");
+        }
+        echo();
+    }
+    private static void delete(String line){
+        try {
+            int index = Integer.parseInt(line.substring(7).trim()) - 1;
```

Comment:
Avoid magic number. 7 can be confusing for reader who do know what it represents or why is it chosen